packages/eslint-config-airbnb/index.js

```diff
@@ -1,13 +1,7 @@
-const reactRules = require('./react');
-const base = require('./base');
-
-// clone this so we aren't mutating a module
-const eslintrc = JSON.parse(JSON.stringify(base));
-
-// manually merge in React rules
-eslintrc.plugins = reactRules.plugins;
-Object.keys(reactRules.rules).forEach(function assignRule(ruleId) {
-  eslintrc.rules[ruleId] = reactRules.rules[ruleId];
-});
-
-module.exports = eslintrc;
+module.exports = {
+  'extends': [
+    // Need to qualify these here for ESLint to resolve them properly.
```

Comment:
I believe all `extends` in `eslint` have to be named starting with "eslint-config", or a scoped package with the name ending in "/eslint-config".